src/layouts/Titled/components/TOC.tsx

```diff
@@ -53,10 +53,11 @@ export const TOCBlock: React.FC<{ data: IHeading[] } & HTMLAttributes<HTMLDivEle
 
 const StyledAnchor = styled(Anchor)<{ isCurrent: boolean }>`
   display: block;
-  margin: ${p => p.theme.space.xxs} 0;
+  margin: ${p => p.theme.space.base * 0.5}px 0;
```

Comment:
I believe this reduced the touch-target below WCAG AA levels. Did you see any issues when running the Chrome auditing checklist item?